download-results.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/env python3
+
+# Script for downloading results from the OpenScPCA-nf workflow results bucket
+
+import argparse
+import datetime
+import os
+import pathlib
+import re
+import subprocess
+import sys
+from typing import List, Set
+
+# import functions from download-data.py
+download_data = __import__("download-data")
+
+# enable text formatting on Windows
+os.system("")
+
+RESULTS_BUCKET = "openscpca-nf-workflow-results"
+TEST_RESULTS_BUCKET = "openscpca-test-workflow-results-public-access"
+
+
+def get_results_modules(bucket: str, release: str, profile: str) -> List[str]:
+    """
+    Get the list of available results modules from the OpenScPCA workflow results bucket.
+    """
+    ls_cmd = ["aws", "s3", "ls", f"s3://{bucket}/{release}/"]
+    if profile:
+        ls_cmd += ["--profile", profile]
+    if bucket == TEST_RESULTS_BUCKET:
+        ls_cmd += ["--no-sign-request"]
+    ls_result = subprocess.run(ls_cmd, capture_output=True, text=True)
+    if ls_result.returncode:  # authentication errors, usually
+        print(
+            "Error listing release versions from the OpenScPCA bucket.\n\n"
```

Comment:
```suggestion
            "Error listing modules available in OpenScPCA workflow results bucket.\n\n"
```
I think this should be updated to reflect that you are listing modules?